### DISCUSSION TITLE
Plugin to remove prayer filter button

### DISCUSSION THREAD

**Discussion by Autoswitch** [runelite/runelite#16883] [Ideas]
Please add a feature that hides the "Filters" on the prayer/spellbook with the option to hide either as this can be missclicked sometimes resulting in death. An example can be prayer flicking augury but missclicking the filters, you are now sent to a different interface and cannot pray switch to range/mage on time if fighting a boss that uses different combat styles. 

![Screenshot 2023-08-04 134026](https://github.com/runelite/runelite/assets/141421537/e5ffda41-5fb0-4a2a-a5a0-e6f736fcbdb0)

You are getting attacked in the wilderness and would get away from the opponent if you freeze them but instead missclick the spellbook filters. Yes, skill issue etc. but if this button didn't exist, the missclick would only cost a tick or two, but instead having to close the entire interface to get back to the spellbook is a pain.

**Comment by Felanbird** [comment]
![java_2023-08-04_14-42-38](https://github.com/runelite/runelite/assets/41973452/229bb70c-d440-490f-9ab9-68aeb654e1a4)
